RNWCPP/ReactWindowsCore/Modules/ExceptionsManagerModule.cpp

```diff
@@ -80,9 +80,9 @@ JSExceptionInfo ExceptionsManagerModule::CreateExceptionInfo(const folly::dynami
   for (const auto &stackFrame : stackAsFolly)
   {
     // Each dynamic object is a map containing information about the stack frame:
-    // method (string), filename(string), line number (int) and column number (int).
+    // method (string), arguments (array), filename(string), line number (int) and column number (int).
     assert(stackFrame.type() == folly::dynamic::OBJECT);
-    assert(stackFrame.size() == 4);
+    assert(stackFrame.size() == 5);
```

Comment:
>assert(stackFrame.size() == 5); [](start = 4, length = 31)

I had fixed this previously but Hal's OACR changes backed out my change, arg!

Reply:
Yeah, after I submitted the PR I dug I got curious and dug into the history and found your change and the OACR change.